dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/plugins/custom/OauthClientStore.java

```diff
@@ -321,7 +322,8 @@ void logMessageAndThrowOidcServerException(OAuthStoreException e, String msgKey,
         // Use an empty string as the last insert to avoid an unused insert (e.g. "{2}") showing up in the exception message
         updatedMsgArgs = appendStringMessageToArgs("", msgArgs);
         String exceptionMsg = Tr.formatMessage(tc, msgKey, updatedMsgArgs);
-        throw new OidcServerException(exceptionMsg, OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e);
+        throw new OidcServerException(new BrowserAndServerLogMessage(tc, null, exceptionMsg), OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e);
```

Comment:
The `exceptionMsg` variable is the already translated message, which isn't what we want yet. We want the `BrowserAndServerLogMessage` object we create to be created using
```Java
new BrowserAndServerLogMessage(tc, null, msgKey, updatedMsgArgs)
```